practice/src/scripts/views/task.view.ts

```diff
@@ -48,9 +75,48 @@ class TaskView {
         }
     };
 
+    handleTaskClose = (event: MouseEvent): void => {
+        const taskId = (event.currentTarget as HTMLElement).getAttribute('data-task-id');
+
+        if (taskId) {
+            this.showConfirmDialog();
+            this.currentTaskId = Number(taskId);
+        }
+    };
+
+    handleConfirmYes = (): void => {
+        if (this.currentTaskId !== null) {
+            this.onTaskClosed(this.currentTaskId);
+            this.hideConfirmDialog();
+            this.currentTaskId = null;
+        }
+    };
+
+    handleConfirmCancel = (): void => {
+        this.hideConfirmDialog();
+
+        this.currentTaskId = null;
+    };
+
+    showConfirmDialog = (): void => {
```

Comment:
showConfirmDialog and hideConfirmDialog can you handle for both in a function toggleConfirmDialog()